Cargo.toml

```diff
@@ -1,8 +1,11 @@
 [package]
 name = "vm-allocator"
 version = "0.1.0"
+description = "Helpers for allocating resources needed during the lifetime of a VM."
```

Comment:
Hmm, just curious, are these actually required before publishing? (i.e. it won't work without them)

Reply:
I am not sure about this particular one, but all rust-vmm crates have it.

Reply:
Yes the `description` is required: https://doc.rust-lang.org/cargo/reference/manifest.html#the-description-field.